src/main/java/com/google/sps/servlets/CasesDataServlet.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+   Builds and returns case data as a JSON array, e.g.
+    [{"lat": 38.4404675, "lng": -122.7144313, "active": 5}]
+ */
+@WebServlet("/report")
+public class CasesDataServlet extends HttpServlet {
+  private String reportsJson;
+  public static final String STATS = "/WEB-INF/cases.csv"; // COVID-19 case data
+  public static final String CTYPE = "application/json"; // HttpServletResponse content type
+  public static final String ENCODING = "UTF-8"; // HttpServletResponse character encoding
+
+  // Builds Json array using data set
```

Comment:
For method, it's better to use javadoc style comment: https://en.wikipedia.org/wiki/Javadoc